### DISCUSSION TITLE
Upgrade to recent xorm 1.x

### DISCUSSION THREAD

**Discussion by lunny** [grafana/grafana#48097] [Platform]
Looks like grafana is still using xorm 0.8.2, I think it's time to upgrade to 1.x

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
